packages/core/src/prompts/text.ts

```diff
@@ -1,7 +1,7 @@
 import color from 'picocolors';
 import Prompt, { type PromptOptions } from './prompt.js';
 
-export interface TextOptions extends PromptOptions<TextPrompt> {
+interface TextOptions extends PromptOptions<TextPrompt> {
```

Comment:
these don't get exported from `src/index.ts` currently. we need to export all the `Options` from all the prompts in a separate PR